tasks/eth/006-ecotone-set-gas-config/README.md

```diff
@@ -0,0 +1,76 @@
+# Mainnet - Ecotone Blobs Gas Configuration
+
+Status: DRAFT
+
+## Objective
+
+This is the playbook for updating the gas config of the `SystemConfig` to be ready to send
+blob batch transactions.
+
+The proposal was:
+- [ ] approved by Token House voting here: TODO 
+- [ ] not vetoed by the Citizens' house here: TODO
+
+Following [internal calculations](https://github.com/ethereum-optimism/protocol-quest/issues/84#issuecomment-1977330679),
+the basefee and blob basefee scalars are updated to
+* `L1BaseFeeScalar: 1353`
+* `BlobBaseFeeScalar: 810945`
```

Comment:
Is there something public we can share here? What justification did we show to governance for these values?

Reply:
This is based on calculations by @K-Ho and @MSilb7 - will ask them what we can provide here. Maybe we share the rationale behind this internally first for the purpose of this singing ceremony, and explain where these values come from in a later public post?

Reply:
Hi yeah actually 2 things:
1. [This sheet (which should be public)](https://docs.google.com/spreadsheets/d/12VIiXHaVECG2RUunDSVJpn67IQp9NHFJqUsma2PndpE/edit#gid=186414307) - Configured to:
- Transactions per day: 360,000
- Transaction Type : OP Mainnet
- op-batcher Max Channel Duration (hours): 5
- Target Data Margin: 5%

2. I can make a comment elsewhere too, but now handling for "batch overhead" and "blob overhead" slightly reducing the available bytes per blob of transaction data, scalars by this calculator should be:
* `L1BaseFeeScalar: 1367`
* `BlobBaseFeeScalar: 810949`

Super super tiny mods, but likely worth updating. I'll update in https://github.com/ethereum-optimism/protocol-quest/issues/84#issuecomment-1977330679 as well

Reply:
^ updated the L1BaseFeeScalar to match the sheet

Reply:
I could do a quick dev blog on how we calculated if needed as well. Could be good content as part of a series on "where fees come from," and "how we solve/handle for various forms of congestion" which I've been thinking about.

Reply:
Note that the calculator now shows `810949` and `1368` (1 higher). These values are also in the internal issue.

Reply:
updated readme to mention public calculator and new scalar values.